application/shared-kernel/ApiCore/ApiCoreConfiguration.cs

```diff
@@ -164,6 +179,14 @@ public static WebApplication AddApiCoreConfiguration<TDbContext>(this WebApplica
 
         // Add test-specific endpoints when running tests, such as /api/throwException
         app.MapTestEndpoints();
```

Comment:
I guess the `TestEndpoints`, `HealtEndpoints`, and `TrackEndpoints` in the ApiCore, could/should also be implemented using the `IEndpoints`. We

The `HealtEndpoints` are in the `ServiceDefaultsExtensions` class. We should extract these to a separate class and move them to the `/Endpoints` folder.

Reply:
Done